client/my-sites/themes/themes-selection.jsx

```diff
@@ -30,6 +31,9 @@ import { setThemePreviewOptions } from 'state/themes/actions';
 import config from 'config';
 import { PAGINATION_QUERY_KEYS } from 'lib/query-manager/paginated/constants';
 
+const trackThemeClick = recordTracksEvent.bind( null,
+	'calypso_themeshowcase_theme_click' );
+
```

Comment:
I'd remove the binding and just `connect` `recordTracksEvent` (and call it from inside the component with that event name string arg passed in). Rationale: We currently use `analytics.tracks.recordEvent` in a couple of other spots (with different even names) inside this component, so this would enable us to re-use the action if we ever wanted to.